src/components/InputContainer/styles.module.css

```diff
@@ -2,33 +2,30 @@
     position: relative;
 
     .input-section {
-        display: flex;
-        align-items: center;
+        flex-wrap: nowrap;
         transition: var(--go-ui-duration-transition-medium) var(--go-ui-color-gray-10) ease-in-out;
-        border: var(--go-ui-width-separator-thin) solid transparent;
+        border-bottom: var(--go-ui-width-separator-thin) solid transparent;
         border-radius: var(--go-ui-border-radius-md);
         background-color: var(--go-ui-color-gray-10);
-        padding: var(--go-ui-spacing-sm) var(--go-ui-spacing-md);
-
-        .internal-input-container {
-            flex-grow: 1;
-
-            &.disabled {
-                cursor: not-allowed;
-            }
-        }
+        padding: var(--go-ui-spacing-xs) var(--go-ui-spacing-md);
```

Comment:
![image](https://github.com/IFRCGo/go-web-app/assets/7893887/c5760436-4b3c-44e8-bc5e-72ef4e861f4b)
Padding is reduced significantly. Was this intentional?